src/AboutContent.js

```diff
@@ -9,9 +12,36 @@ const AboutContent = function ( props ) {
     <wrapper>
       
       <Header as='h1' style={{ fontSize: '4em' }}>About the Cliff Effects Tool</Header>
-      <p style={{ fontSize: '1.33em' }}>
-        Fill in information about the tool here.
-      </p>
+
+      <Header as='h3'>What is this?</Header>
+      <p>This tool can help show how a change in income affects how much someone receives in public assistance from SNAP and Section 8 Housing Voucher benefits. It was designed for the case managers at <ExternalLink href='http://www.prohope.org/'>Project Hope</ExternalLink> with the aim of helping to predict changes in their client's benefits.</p>
+
+      <Header as='h3'>Why are you doing this?</Header>
```

Comment:
Bit of an odd phrasing. Sounds like what the hero says in a plotline when they're backstabbed by a team member. I'd suggest the less pointed 'Why is this important?'.

Reply:
"Why is this tool important?"